SecretSanta/src/SecretSanta.Api/Controllers/UsersController.cs

```diff
@@ -15,24 +18,39 @@ public class UsersController : ControllerBase
 
         public UsersController(IUserRepository repository)
         {
-            Repository = repository ?? throw new System.ArgumentNullException(nameof(repository));
+            Repository = repository ?? throw new ArgumentNullException(nameof(repository));
         }
 
         [HttpGet]
-        public IEnumerable<User> Get()
+        [ProducesResponseType(typeof(IEnumerable<User>), StatusCodes.Status200OK)]
```

Comment:
This is indicating the wrong response type. 
```suggestion
        [ProducesResponseType(typeof(IEnumerable<DtoUser>), StatusCodes.Status200OK)]
```